cmd/util.go

```diff
@@ -47,3 +48,11 @@ func parseRepoAndPathMaybe(repoAndPath string) (string, string, error) {
 		repoAndPath)
 
 }
+
+func PrintValues(values []string, ignoreValue string){
```

Comment:
not a request but just a comment on this. One of the best practices of go when dealing with I/o is that functions that read and/or read don't assume a specific read or write source (like a file or socket or std(in|out). The caller determines this. So if we were to follow this convention PrintValues method sig might look like PrintValues(w io.Writer, values []string,ignoreValue string) so that where the output goes is determined by caller and function implementation doesn't care.  the interface io.REader and io.Writer are used pervasively this way in go . Have I always done this? Probably not but I respect and see value in the practice. Food for thought